FastSimulation/Tracking/python/IterativePixelLessStep_cff.py

```diff
@@ -33,6 +33,8 @@
 #                                     'TEC1_pos+TEC2_pos','TEC2_pos+TEC3_pos','TEC3_pos+TEC4_pos','TEC3_pos+TEC5_pos','TEC4_pos+TEC5_pos',
 #                                     'TID1_neg+TID2_neg','TID2_neg+TID3_neg',
 #                                     'TEC1_neg+TEC2_neg','TEC2_neg+TEC3_neg','TEC3_neg+TEC4_neg','TEC3_neg+TEC5_neg','TEC4_neg+TEC5_neg']
+
+from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
```

Comment:
in the following line you import pixelLessStepSeedLayers that makes use of it
There is most probably a different way to import it silently, at the moment my expertise in python (and understanding of RefToPset) makes the current solution the easier